examples/glyph-vector.py

```diff
@@ -5,6 +5,12 @@
 #  FreeType high-level python API - Copyright 2011 Nicolas P. Rougier
 #  Distributed under the terms of the new BSD license.
 #
+#  - The code is strickly speaking not optimal, as it ignores the 3rd order
```

Comment:
'not optimal' isn't quite the right word - on rethinking, I think 'incomplete' might be better regarding the interpolation and assuming first point is on. The spltting 3rd-order curve into two 2nd order one is strictly speaking a bug, but can only be seen if you look at a postscript font (such as those shipped with adobe acrobat reader - see latest figures on https://github.com/rougier/freetype-py/issues/58 ).